mlir/test/lit.local.cfg

```diff
@@ -0,0 +1,7 @@
+if not "SPIRV" in config.root.targets:
+    config.unsupported = True
```

Comment:
Why do we need the SPIRV backend to run MLIR unit-tests?

Reply:
Please see this comment: https://github.com/llvm/llvm-project/pull/143108#issuecomment-2963067688

Reply:
I missed this -- if this disables all mlir tests, we should revert the PR

Reply:
Revert PR: https://github.com/llvm/llvm-project/pull/144773